packages/dcos-integration-test/extra/conftest.py

```diff
@@ -29,6 +29,7 @@ def cluster():
     assert 'SLAVE_HOSTS' in os.environ
     assert 'PUBLIC_SLAVE_HOSTS' in os.environ
     assert 'DNS_SEARCH' in os.environ
+    assert 'DOCKERCFG_HOOK' in os.environ
```

Comment:
Is this applicable to Azure and AWS? If not, I would prefer this to have a false default instead of assert on unset